src/components/views/messages/BridgeError.js

```diff
@@ -67,56 +87,98 @@ export default class BridgeError extends React.PureComponent {
         return members.filter(m => regex.test(m.userId));
     }
 
+    /**
+     * Sanitized infos from a relation.
+     * @typedef {Object} RelationInfo
+     * @property {RoomMember[]} affectedUsers
+     * @property {string} eventID
+     * @property {string} networkName
+     * @property {string} reason
+     */
+
     /**
      * Returns the network name and the affected users for the given relation.
      *
      * @param {MatrixEvent} relation
-     * @returns {{networkName: string, affectedUsers: RoomMember[]}}
+     * @returns {RelationInfo}
      */
     _getRelationInfo(relation) {
         if (!relation.event || !relation.event.content) {
             return { networkName: "", affectedUsers: [] };
         }
         const content = relation.event.content;
 
-        const networkName = (typeof content.network_name === 'string' ?
-            content.network_name :
-            ""
-        );
-        const affectedUsersRegex = (Array.isArray(content.affected_users) ?
-            content.affected_users :
-            []
-        );
+        const affectedUsersRegex = assureArray(content.affected_users);
         const affectedUsers = affectedUsersRegex.flatMap(u =>
             this._findMembersFromRegex(u),
         );
 
-        return { networkName, affectedUsers };
+        return {
+            affectedUsers: affectedUsers,
+            eventID: assureString(content.event_id),
+            networkName: assureString(content.network_name),
+            reason: assureString(content.reason),
+        };
+    }
+
+    _errorMessages = {
+        "m.event_not_handled": _td(
+            "⚠ Not delivered to people on %(networkName)s (%(affectedUsers)s)",
+        ),
+        "m.event_too_old": _td(
+            "⚠ It took so long. Gave up sending to people on %(networkName)s " +
+            "(%(affectedUsers)s)",
+        ),
+        "m.internal_error": _td(
+            "⚠ Unexpected error while sending to people on %(networkName)s " +
+            "(%(affectedUsers)s)",
+        ),
+        "m.foreign_network_error": _td(
+            "⚠ %(networkName)s did not deliver the message to the people " +
+            "there (%(affectedUsers)s)",
+        ),
+        "m.event_unknown": _td(
+            "⚠ Was not understood by %(networkName)s, so people there didn't " +
```

Comment:
I have to agree with @bwindels about using the existing warning icon, sorry. Means pulling the emoji out of the string, but using our existing assets for images.

Reply:
I am on it already :smile: